### PR TITLE
Enhancement code style check

### PR DESCRIPTION
**Pull request by wuchunfu**
---

## Check list

* [ ] Code changed are covered with tests, or it does not need tests for reason:
* [ ] Change does not need document change, or I will submit document change to https://github.com/InterestingLab/seatunnel-docs later

### CODE REVIEW

File: plugin-flink-source-file/src/main/java/io/github/interestinglab/waterdrop/flink/source/TextRowInputFormat.java [hunk 1]
```diff
@@ -65,9 +65,7 @@ public void configure(Configuration parameters) {
 
     @Override
     public Row readRecord(Row reusable, byte[] bytes, int offset, int numBytes) throws IOException {
-        if (this.getDelimiter() != null && this.getDelimiter().length == 1
-                && this.getDelimiter()[0] == NEW_LINE && offset + numBytes >= 1
-                && bytes[offset + numBytes - 1] == CARRIAGE_RETURN) {
+        if (this.getDelimiter() != null && this.getDelimiter().length == 1 && this.getDelimiter()[0] == NEW_LINE && offset + numBytes >= 1 && bytes[offset + numBytes - 1] == CARRIAGE_RETURN) {
```
Comment (CalvinKirs) [inline plugin-flink-source-file/src/main/java/io/github/interestinglab/waterdrop/flink/source/TextRowInputFormat.java:68]: TBH,I prefer the previous one, it looks clearer

File: seatunnel-config/src/main/java/io/github/interestinglab/waterdrop/config/ConfigFactory.java [hunk 2]
```diff
@@ -153,19 +153,19 @@ public static Config load(ClassLoader loader, String resourceBasename,
...
     private static ClassLoader checkedContextClassLoader(String methodName) {
         ClassLoader loader = Thread.currentThread().getContextClassLoader();
-        if (loader == null)
-            throw new ConfigException.BugOrBroken("Context class loader is not set for the current thread; "
-                    + "if Thread.currentThread().getContextClassLoader() returns null, you must pass a ClassLoader "
-                    + "explicitly to ConfigFactory." + methodName);
-        else
+        if (loader == null) {
+            throw new ConfigException.BugOrBroken("Context class loader is not set for the current thread; " + "if Thread.currentThread().getContextClassLoader() returns null, you must pass a ClassLoader " + "explicitly to ConfigFactory." + methodName);
+        } else {
```
Comment (CalvinKirs) [inline seatunnel-config/src/main/java/io/github/interestinglab/waterdrop/config/ConfigFactory.java:158]: Here `else` is redundant

File: seatunnel-config/src/main/java/io/github/interestinglab/waterdrop/config/ConfigFactory.java [hunk 3]
```diff
@@ -153,19 +153,19 @@ public static Config load(ClassLoader loader, String resourceBasename,
...
+        }
     }
 
     private static ConfigParseOptions ensureClassLoader(ConfigParseOptions options, String methodName) {
-        if (options.getClassLoader() == null)
+        if (options.getClassLoader() == null) {
             return options.setClassLoader(checkedContextClassLoader(methodName));
-        else
+        } else {
             return options;
```
Comment (CalvinKirs) [inline seatunnel-config/src/main/java/io/github/interestinglab/waterdrop/config/ConfigFactory.java:167]: same question

**Comment by zhongjiajie** [comment]
@wuchunfu NIT, maybe we should change PR title to `Enhancement code style check`

File: tools/checkstyle/checkStyle.xml [hunk 4]
```diff
@@ -17,11 +17,21 @@
   ~
```
Comment (timothy65535) [inline tools/checkstyle/checkStyle.xml:17]: change `checkStyle.xml` to `checkstyle.xml` make sense.

File: plugin-flink-source-file/src/main/java/io/github/interestinglab/waterdrop/flink/source/TextRowInputFormat.java (hunk 1 above)
Comment (wuchunfu) [reply plugin-flink-source-file/src/main/java/io/github/interestinglab/waterdrop/flink/source/TextRowInputFormat.java:68]: OK, I'll adjust it

File: seatunnel-config/src/main/java/io/github/interestinglab/waterdrop/config/ConfigFactory.java (hunk 2 above)
Comment (wuchunfu) [reply seatunnel-config/src/main/java/io/github/interestinglab/waterdrop/config/ConfigFactory.java:158]: OK, I'll adjust it

File: tools/checkstyle/checkStyle.xml [hunk 5]
```diff
@@ -70,50 +297,227 @@
...
+        </module>
+
+        <module name="CatchParameterName">
+            <property name="format" value="^[a-z]([a-zA-Z0-9]*)?$"/>
+            <message key="name.invalidPattern" value="Catch parameter name ''{0}'' must match pattern ''{1}''."/>
         </module>
-        <!--todo -->
-        <!--<module name="PackageName">
-            <property name="format" value="^(org|test)\.apache\.seatunnel(\.[a-zA-Z][a-zA-Z0-9]*)+$"/>
-        </module>-->
```
Comment (CalvinKirs) [inline tools/checkstyle/checkStyle.xml:96]: please keep this

File: seatunnel-config/src/main/java/io/github/interestinglab/waterdrop/config/impl/MemoKey.java [hunk 6]
```diff
@@ -41,21 +41,22 @@ public final int hashCode() {
...
                 return false;
-            else if (o.restrictToChildOrNull == this.restrictToChildOrNull)
+            } else if (o.restrictToChildOrNull == this.restrictToChildOrNull) {
                 return true;
-            else if (o.restrictToChildOrNull == null || this.restrictToChildOrNull == null)
+            } else if (o.restrictToChildOrNull == null || this.restrictToChildOrNull == null) {
                 return false;
-            else
+            } else {
                 return o.restrictToChildOrNull.equals(this.restrictToChildOrNull);
```
Comment (CalvinKirs) [inline seatunnel-config/src/main/java/io/github/interestinglab/waterdrop/config/impl/MemoKey.java:51]: This kind of `else` is meaningless

File: tools/checkstyle/checkStyle.xml (hunk 4 above)
Comment (leo65535) [reply tools/checkstyle/checkStyle.xml:17]: +1, we can use `checkstyle.xml` as the file name.

**Review by CalvinKirs — APPROVED** [review]

**Pull request merged**